site/en/docs/extensions/mv3/declare_permissions/index.md

```diff
@@ -91,13 +90,13 @@ The following table lists the currently available permissions:
     </tr>
     <tr id="clipboardRead">
       <td><code>"clipboardRead"</code></td>
-      <td>Required if the extension or app uses <code>document.execCommand('paste')</code>.</td>
+      <td>Required if the extension uses <code>document.execCommand('paste')</code>.</td>
     </tr>
     <tr id="clipboardWrite">
       <td><code>"clipboardWrite"</code></td>
-      <td>Indicates the extension or app uses <code>document.execCommand('copy')</code> or
-        <code>document.execCommand('cut')</code>. This permission is <b>required for hosted apps</b>; it's recommended
-        for extensions and packaged apps.</td>
+      <td>Indicates the extension uses <code>document.execCommand('copy')</code> or
+        <code>document.execCommand('cut')</code>. This permission is is recommended
+        for all extensions.</td>
```

Comment:
```suggestion
        <code>document.execCommand('cut')</code>.
```

I'm not entirely sure what the motivation here was originally, but I don't think this is recommended for all extensions (I don't see why it would be).